cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -65,6 +65,9 @@ OPTIONS
        --debug
            Run tests in remote debugging mode, only when the '--skip-tests' flag is not used.
 
+       --with-choreo
+           Package Choreo observability extension in the executable jars.
+
 CONFIG PROPERTIES
        (--key=value)...
            Set Ballerina environment parameters as key/value pairs.
```

Comment:
```suggestion
           Set the Ballerina environment parameters as key/value pairs.
```

Reply:
Fixed in 77fd6022bdbaf2870da71ae578be26ded14c77b5